agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -579,6 +584,66 @@ public void validate() throws FriendlyException {
         public GcReportingLevel reportingLevel = GcReportingLevel.TENURED_ONLY;
     }
 
+    public static class AadAuthentication {
+        public AadAuthentication() {
+            this.authenticationEnabled = false;
+        }
+        public boolean authenticationEnabled;
```

Comment:
do we need this, or can we use presence/absence of `type` to determine if it's enabled?

Reply:
this is to avoid exception thrown from the validate function in AadAuthentication class, when the AAD authentication config is not provided by the user